guides/hosting/infrastructure/message-queue.md

```diff
@@ -116,7 +132,7 @@ shopware:
     admin_worker:
         enable_admin_worker: true
         poll_interval: 30
-        transports: ["async"]
+        transports: ["async", "async_low_priority"]
```

Comment:
```suggestion
        transports: ["async", "failed", "async_low_priority"]
```

@keulinho was it just missing or is the admin worker not supposed to consume the `failed` transport? 🤔

Reply:
I wanted to ask the same for the command. In the symfony docs they recommend to run the failed queue manually and have a look at the failed messages :sweat_smile:

Reply:
Yes so it was intentional ;)